### DISCUSSION TITLE
カスタム絵文字をスタンプに変えることについて

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [misskey-dev/misskey#13602] [Ideas]
発端はしゅいろ氏による次のノートです:

> 「カスタム絵文字」という呼称やめて「スタンプ」にするかしら
> そう呼んでる人多いし、カスタムされている絵文字というのは意味が分かりにくいし
> -- https://misskey.io/notes/9r3b89e0tiog00qv

確かにそう呼んでいる人もioでは一定数居るかもしれません (※調べていません)。

しかし、私としては以下の理由から混乱を招くように思います。
1. 他のサービス、例えばLINEやDiscordなどでは絵文字とスタンプの概念は明確に分離されているように思うため
2. カスタム絵文字という呼び方は「Unicodeではない、ユーザーが定義した絵文字」という意味合いがある
3. **私の体感**としてはカスタム絵文字と呼んでいる人のほうがスタンプと呼んでいる人より使ってる人が多い (※要出典)

そのため、Misskeyでカスタム絵文字をスタンプに変えるのは微妙な気がします。

**Comment by samunohito** [comment]
slack、GoogleChatも「カスタム絵文字」と呼称されていますね。
「スタンプ」も「カスタム絵文字」もソフトウェア上の方言に過ぎず、どちらも広く通用する言い回しだと思います。
（これが明らかに意味の通用しない謎の独自用語だったら一考する余地はあるかと思いますが）

なので…変える必要があるかといわれると、不要な気もしており…

**Comment by kakkokari-gtyih** [comment]
カスタム絵文字、変えるとしても、Unicode絵文字とカスタム絵文字をあわせて単に「絵文字」として呼称した上で、サーバー内に存在する絵文字を指すときには「サーバー固有の絵文字」とかにしてもいいのかもしれない

ref: https://misskey.io/notes/9r3by45ayu2z0e46

**Comment by syamatsumi** [comment]
これ「エモートピクチャ」とか「エモート画像」って呼べば実際の利用状況に近い表現になる気がする……
実際、文中においてもエモーショナルな表現に用いられがちだし……

あるいはスタンプと合成して「エモートスタンプ」とか、絵文字要素を残して「絵文字スタンプ」と呼ぶのもアリか？
